source/git-util.js

```diff
@@ -133,6 +133,29 @@ exports.tagExistsOnRemote = async tagName => {
 	}
 };
 
+exports.defaultBranch = async options => {
+	if (options.releaseBranch) {
+		return options.releaseBranch;
+	}
+
+	for (const branch of ['main', 'master', 'gh-pages']) {
+		try {
+			// eslint-disable-next-line no-await-in-loop
+			await execa('git', [
+				'show-ref',
+				'--verify',
+				'--quiet',
+				`refs/heads/${branch}`
+			]);
+			return branch;
+		} catch {}
+	}
+
+	throw new Error(
+		'Could not determine a default branch. Please specify one via --branch.'
```

Comment:
```suggestion
		'Could not infer the default Git branch. Please specify one with the --branch flag.'
```

Reply:
The user could also specify the `branch` config. This should include a mention of that too. https://github.com/sindresorhus/np#config